kouncil-frontend/src/app/track/track-result/track-result.component.ts

```diff
@@ -176,6 +186,9 @@ export class TrackResultComponent implements OnInit, OnDestroy {
             this.allRows = [...this.allRows, ...events];
```

Comment:
imo to co sie tutaj wyrabia, to jest idealny kandydat do refaktoru. Przemieszana logika i hack na changeDetecta

Reply:
to jest temat na osobny refactor ;)